### PR TITLE
Mo/documentation enhancements

### PR DESCRIPTION
**Pull request by MauriceReeves-usds**
This PR updates the documentation and the documentation generator to do the following:

- Include the default value
- Output all the HL7 fields now
- Link to the documentation for the HL7 segments on Carisitix 

## Checklist
- [x] Tested locally?
- [x] Ran `quickTest all`?
- [x] Ran `./prime test` against local Docker ReportStream container?
- [x] Downloaded a file from http://localhost:7071/api/download
- [ ] Updated the release notes? 
- [ ] Added tests?
- [ ] Did you check for sensitive data, and remove any? 
- [ ] Does logging contain sensitive data?  
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

### CODE REVIEW

**Review by jimduff-usds — APPROVED** [review]
@mauricereeves , It looks like there are changes I made to TestReportStream.kt that are showing as diffs in your PR.  I think/hope they'll go away if you do a merge from master. Thanks.

Looks good otherwise.

File: prime-router/settings/organizations-local.yml [hunk 1]
```diff
@@ -208,6 +208,52 @@
...
+        deidentify: false
+        translation:
+          type: HL7
+          useBatchHeaders: true
+          receivingApplicationName: NBS
+          receivingApplicationOID: 2.16.840.1.114222.4.1.185.1
+          receivingFacilityName: VDH
+          receivingFacilityOID: 2.16.840.1.114222.4.1.185
+
   - name: nd-doh
```
Comment (carlosfelix2) [inline prime-router/settings/organizations-local.yml:257]: These changes are beyond documentation.  Is this one change Jim mentioned?

File: prime-router/src/main/kotlin/cli/TestReportStream.kt [hunk 2]
```diff
@@ -527,9 +527,9 @@ Examples:
     }
 
     private fun doBadCsvTest(environment: ReportStreamEnv) {
-        val filenames = listOf("not-a-csv-file.csv", /* "column-headers-only.csv", */ "completely-empty-file.csv")
-        filenames.forEachIndexed { i, filename ->
-            ugly("Starting badcsv file Test $i: submitting $filename")
+        val filenames = listOf("not-a-csv-file.csv", "column-headers-only.csv", "completely-empty-file.csv")
```
Comment (carlosfelix2) [inline prime-router/src/main/kotlin/cli/TestReportStream.kt:530]: These changes are also beyond documentation.  Are these supposed to be here?

**Review by carlosfelix2 — COMMENTED** [review]
There seem to be more than document changes here.  Do those extra changes need to be removed?

**Pull request merged**